plugins/webp-uploads/settings.php

```diff
@@ -178,6 +200,94 @@ function webp_uploads_generate_webp_jpeg_setting_callback(): void {
 	<?php
 }
 
+
+/**
+ * Renders the settings field for generating all fallback image sizes.
+ *
+ * @since n.e.x.t
+ */
+function webp_uploads_generate_all_fallback_sizes_callback(): void {
+	$all_fallback_sizes_enabled   = 1 === (int) get_option( 'perflab_generate_all_fallback_sizes', 0 );
```

Comment:
```suggestion
	$all_fallback_sizes_enabled   = 1 === webp_uploads_should_generate_all_fallback_sizes();
```

Can we re-use `webp_uploads_should_generate_all_fallback_sizes()` here as well?

Reply:
Yes we can done in https://github.com/WordPress/performance/pull/1689/commits/3a647137c35afd2d2fe57d7af7a80c653d3defa4